src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/AbstractConditionMapper.java

```diff
@@ -0,0 +1,355 @@
+/*
```

Comment:
The key condition mapping logic is basically the same as what we had in HashPartitioningConditionMapper before. The update expression mapping logic is new. Both strategies now share this base class.